ax/service/ax_client.py

```diff
@@ -1765,6 +1787,8 @@ def _gen_new_generator_run(self, n: int = 1) -> GeneratorRun:
                 parameters={}, trial_index=self._get_last_completed_trial_index()
             )
         )
+        if fixed_features:
+            fixed_feats.update_features(fixed_features)
```

Comment:
@Cesar-Cardoso This is really the only thing that's changing. The rest is plumbing to be able to pass `fixed_features` down to `_gen_new_generator_run(...)`.